Source/Core/Editor/ProcessUpgradeTool/ProcessUpgradeTool.cs

```diff
@@ -60,85 +54,73 @@ public static IEnumerable<IConverter> Converters
-                "Proceed?", "Yes", "No"))
-            {
-                UpdateObjectGroups();
-            }
-        }
-
-        private static void UpdateObjectGroups()
-        {
-            int counter = 0;
-#pragma warning disable CS0618 // Type or member is obsolete
-            foreach (SceneObjectTags.Tag tag in SceneObjectTags.Instance.Tags)
-            {
-                if (SceneObjectGroups.Instance.GroupExists(tag.Guid) == false)
-                {
-                    SceneObjectGroups.Instance.CreateGroup(tag.Label, tag.Guid);
-                    counter++;
-                }
-            }
-#pragma warning restore CS0618 // Type or member is obsolete
-
-            if (counter > 0)
-            {
-                EditorUtility.SetDirty(SceneObjectGroups.Instance);
-            }
-
-            UnityEngine.Debug.Log($"Converted {counter} tags to object groups.");
-        }
-
-        [MenuItem("Tools/VR Builder/Developer/Update Process in Scene", false, 70)]
-        private static void UpdateProcessMenuEntry()
-        {
-            if (RuntimeConfigurator.Exists == false)
-            {
-                UnityEngine.Debug.LogError("This is not a VR Builder scene");
-                return;
-            }
-
-            IProcess process = GlobalEditorHandler.GetCurrentProcess();
-
-            if (process == null)
-            {
-                UnityEngine.Debug.LogError("No active process found.");
-                return;
-            }
-
-            if (EditorUtility.DisplayDialog("Process Upgrade Tool", $"Updating the current process to the newest version of VR Builder.\n" +
-                $"The correct scene needs to be opened for the upgrade to work properly.\n\n" +
-                $"Process: {process.Data.Name}\n" +
-                $"Scene: {SceneManager.GetActiveScene().name}\n\n" +
-                $"Ensure you have backed up your process file before proceeding!\n" +
-                $"Continue?", "Ok", "Cancel") == false)
-            {
-                return;
-            }
-
-            UpdateObjectGroups();
-
-            IEnumerable<ProcessSceneObject> processSceneObjects = SceneUtils.GetActiveAndInactiveComponents<ProcessSceneObject>();
-            foreach (ProcessSceneObject sceneObject in processSceneObjects)
-            {
-                sceneObject.ResetUniqueId();
-#pragma warning disable CS0618 // Type or member is obsolete
-                foreach (Guid guid in sceneObject.Tags)
-                {
-                    sceneObject.AddGuid(guid);
-                }
-#pragma warning restore CS0618 // Type or member is obsolete
-
-                EditorUtility.SetDirty(sceneObject);
-            }
-
-            UpdateDataOwnerRecursively(process);
-
-            ProcessAssetManager.Save(process);
-        }
+        //[MenuItem("Tools/VR Builder/Developer/Fix Groups", false, 70)]
```

Comment:
I'm suggesting to only comment out the MenuItem here and further down and leave the code.
When e.g. renaming something the code her will other wiese nit be updated.
A short <summary> would also be helpfull here.

Reply:
Good point, done.